prow/plugins/label/label.go

```diff
@@ -44,7 +44,7 @@ func helpProvider(config *plugins.Configuration, enabledRepos []string) (*plugin
 	// The Config field is omitted because this plugin is not configurable.
 	return &pluginhelp.PluginHelp{
 			Description: "The label plugin provides commands that add or remove certain types of labels. Labels of the following types can be manipulated: 'area/*', 'priority/*', 'kind/*', and 'sig/*'.",
-			WhoCanUse:   "Members of the repository's organization may use the label commands.",
+			WhoCanUse:   "Anyone can trigger this plugin on a PR.",
```

Comment:
Thanks for fixing this. I saw `IsMember` in the githubClient interface and assumed it was used :laughing: